pipeline/main.py

```diff
@@ -0,0 +1,37 @@
+"""Run analyses for FinRegistry data"""
```

Comment:
I am a bit confused here about this file fits with the config and analysis scripts and probably need some explaining :sweat_smile: .

- It appears this is FinRegistry specific?
- It is running the survival analysis only? I am guessing it is because you will focus on that first. However, would that code be better off in the survival analysis file rather than in main.py?
- The config.py doesn't seem to be used?

Reply:
Ehh me too, to be honest :D I wrote `main.py` to solve my problem of "I don’t actually know how the whole thing should be run yet".

Following the [DS project organization](https://gist.github.com/ericmjl/27e50331f24db3e8f957d1fe7bbbe510) everything under `risteys_pipeline` is a function, so I wrote `main.py` to run the analyses (i.e. call the functions in risteys_pipeline). Following the same guide, I think the correct place for this would actually be in `scripts/`. 

My end goal was to build a pipeline where each analysis could be run as part of a pipeline where data is preprocessed only once and stored in temporary feather files so that the same preprocessed data could be used by e.g. both "cumulative incidence" and "mortality analysis". However, based on this I should probably implement something for running each analysis independently first.

So in conclusion, I'll move all this someplace else and remove the `main.py`.

Reply:
Moved the contents of the `main.py` to `scripts/finregistry_mortality_analysis.py` (for now, at least): c5a8277

Reply:
Sounds good :slightly_smiling_face: !